sorting.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+import random
+
+
+def insertion_sort(list):
+    for i in range(1, len(list)):
+        key_value = list[i]
+        scan_pos = i - 1
+        while (scan_pos >= 0) and (list[scan_pos] > key_value):
+            list[scan_pos + 1] = list[scan_pos]
+            scan_pos = scan_pos - 1
+        list[scan_pos + 1] = key_value
+
+
+def print_list(list):
+    for item in list:
+        print("{:3}".format(item), end="")
+    print()
+
+list = []
+for i in range(10):
+    list.append(random.randrange(100))
```

Comment:
This can be done with
```for i in random.sample(range(100), 10):
    list.append(i)
```
Or I am pretty sure..
```list.append(random.sample(range(100), 10)```